### DISCUSSION TITLE
hplip

### DISCUSSION THREAD

**Discussion by constantius24** [mocaccinoOS/mocaccino#136] [General] [answered]
is there hplip driver available in the system? i cant find it in any repo

**Comment by joostruis** [comment, marked as answer]
I've added apps/hplip to the community repository.

**Comment by joostruis** [comment]
Did you manage to install and test it?

**Reply by wvpianoman** [reply]
I just tied with the new plasma iso.  I believe we also need hplip-gui to get it to work correctly
